### ISSUE TITLE
Global ids from objects with composite primary keys are not correctly extracted (again)

### ISSUE THREAD

**Issue by genericmoniker** [graphql-python/graphene-sqlalchemy#388]
In #43 a change was made to convert composite keys to a tuple.

graphql-core [made a change](https://github.com/graphql-python/graphql-core/commit/bc66414922dceb33600a7225fb54b362127ed366#diff-582554e849c3ddef07f16a02058d7a2f3b4eb7282c171eddc953889366098d57) such that a tuple is no longer accepted as a valid ID (whereas it would previously just call `str` on the tuple).

The result is an error such as the following when an object of the type is requested with its `id` field:

```
ID cannot represent value: (<UUID instance>, <UUID instance>)
```

**Comment by erikwrede** [comment]
Thanks for bringing this up! The easiest fix would most likely be to convert tuples to strings manually here:
https://github.com/graphql-python/graphene-sqlalchemy/blob/2ca659a7840635a6058f032b9c00488534a07820/graphene_sqlalchemy/types.py#L382-L385

I'll check with the team to make sure this has no other implications and get back to you!

**Comment by bueltan** [comment]
Why not: ? 
`def resolve_id(self, info):
        # graphene_type = info.parent_type.graphene_type
        keys = self.__mapper__.primary_key_from_instance(self)
        return keys[0]`

The current version 3.0.04b has you fixed, but it still failed.

**Comment by erikwrede** [comment]
sorry for the long wait, this was fixed in #399
